### ISSUE TITLE
Requirement for `--solvent_mask` and `--solvent_correct_fsc` in relion_refine?

### ISSUE THREAD

**Issue by olibclarke** [IsoNet-cryoET/spIsoNet#8]
Hi, in the misalignment correction section of the tutorial, it states:

<img width="641" alt="image" src="https://github.com/IsoNet-cryoET/spIsoNet/assets/19766818/065592e5-b2fc-4149-a5f3-afa5cf5e206a">

Why are the latter two arguments needed? I am guessing for calculating the 3D-FSC? We usually only use a solvent mask in relion once we get to local angular searches, as using it during global search can lead to overfitting near the mask edges - but in this case, from what I glean from the preprint, it seems like we will want global search, as the alignment errors for anisotropic data can be quite dramatic. What is the best way to proceed?

As a sidenote re the manuscript, it might be informative to see quantification of per particle alignment errors with/without misalignment correction on, for a case where you have the ground truth alignments.

**Comment by procyontao** [comment]
You are right, mask may not be crucial here. There are two minor reasons: 1. 3DFSC need a mask (maybe for more accurate estimation?). 2. spIsoNet currently recognizes xx_unfil.mrc, which only generated with solvent corrected fsc. I think what you can do currently is to use a really big and smooth mask.

Quantifying per particle alignment error is a good suggestion, I will see what we can find.

**Comment by donghuachensu** [comment]
I remember I often do not use --solvent_correct_fsc in regular Relion auto-refine and it still generates xx_unfil.mrc files. So I don't fully understand your above statement "spIsoNet currently recognizes xx_unfil.mrc, which only generated with solvent corrected fsc". Does it mean that spIsoNet's xx_unfil.mrc files are only generated with --solvent_correct_fsc? Apparently Relion itself does not depend on --solvent_correct_fsc to generate xx_unfil.mrc files (please correct me if wrong).

**Comment by procyontao** [comment]
Hi,

I remember that RELION typically generates the xx_unfil.mrc file after the final iteration. But with the --solvent_correct_fsc flag, it generates the xx_unfil.mrc for each iteration, which is used by spIsoNet. Does that make sense to you?



> Hi, I remember I often do not use --solvent_correct_fsc in regular Relion auto-refine and it still generates xx_unfil.mrc files. So I don't fully understand your above statement "spIsoNet currently recognizes xx_unfil.mrc, which only generated with solvent corrected fsc". Does it mean that spIsoNet's xx_unfil.mrc files are only generated with --solvent_correct_fsc? Apparently Relion itself does not depend on --solvent_correct_fsc to generate xx_unfil.mrc files (please correct me if wrong).
